src/script/builder/tir/prim_func_frame.h

```diff
@@ -55,7 +59,25 @@ class PrimFuncFrame : public TIRFrame {
 
 PrimFuncFrame PrimFunc_(String name);
 tvm::tir::Var Arg(String name, tvm::tir::Var var);
-tvm::tir::Buffer Arg(String name, tvm::tir::Buffer buffer);
+tvm::tir::Var Arg(String name, tvm::tir::Buffer buffer);
+DictAttrs FuncAttrs(DictAttrs attrs);
+tvm::Type FuncRet(tvm::Type ret_type);
+
+tvm::tir::Buffer MatchBuffer(ObjectRef param, Array<PrimExpr> shape,
+                             DataType dtype = DataType::Float(32),
+                             tvm::tir::Var data = NullValue<tvm::tir::Var>(),
+                             Array<PrimExpr> strides = {}, PrimExpr elem_offset = PrimExpr(),
+                             String storage_scope = "", int align = -1, int offset_factor = 0,
+                             String buffer_type_str = "default", Array<IntImm> axis_separators = {},
+                             Span span = Span());
+
+void PreflattenedBuffer(tvm::tir::Buffer postflattened_buffer, Array<PrimExpr> shape,
+                        DataType dtype = DataType::Float(32),
+                        tvm::tir::Var data = NullValue<tvm::tir::Var>(),
```

Comment:
ditto. see: https://github.com/junrushao1994/tvm/pull/42/files#r895045802

```suggestion
                        Optional<tvm::tir::Var> data = NullOpt,
```